ProcessMaker/Nayra/Repositories/ProcessRequestTokenRepository.php

```diff
@@ -74,17 +71,9 @@ public function create(array $transaction): ? Model
      */
     public function update(array $transaction): ? Model
     {
-        // Get the id mapped
-        $id = $this->resolveId($transaction['id']);
-
-        // If not exists throws an error
-        if (!$id) {
-            throw new Exception("Cannot find id for uid {$transaction['id']}");
-        }
-
         // Update the request token
         $properties = $transaction['properties'];
-        $model = ProcessRequestToken::find($id);
+        $model = ProcessRequestToken::find($transaction['id']);
```

Comment:
Is this right? `$transaction['id']` must be the uuid of the request token, not the `id`.
Are we going to use `id` for updates?
Take in count that the engine could create new token and that will have an uuid in Nayra side...